kratos/processes/simple_mortar_mapper_process.cpp

```diff
@@ -46,6 +46,7 @@ SimpleMortarMapperProcess<TDim, TNumNodes, TVarType, TNumNodesMaster>::SimpleMor
     this->Set(MODIFIED, !using_average_nodal_normal);
 
     // We set some values
+    mDiscontinuousInterface = mThisParameters["compute_discontinuous_interface"].GetBool();
```

Comment:
Are you computing the interface? If not the name seems to me a bit misleading since as far as I understand it should be provided.

Reply:
The interface is provided, the parameter can be renamed to be less misleading